tests/ui/UnreadIndicatorsTest.js

```diff
@@ -196,7 +196,7 @@ function signInAndGetAppWithUnreadChat() {
         });
 }
 
-describe('Unread Indicators', () => {
+xdescribe('Unread Indicators', () => {
```

Comment:
We agreed to fix this test suite later if this would be too much of a blocker. Should I remove the `x` anyway? I am not sure what would be the preferred way of handling this situation.